### PR TITLE
Update for Swift 3.0

### PR DESCRIPTION
**Pull request by keith**

### CODE REVIEW

File: Sources/Sonar/APIs/AppleRadarRouter.swift [hunk 1]
```diff
@@ -82,41 +81,40 @@ enum AppleRadarRouter {
...
     }
 
     /// The request representation of the route including parameters and HTTP method.
-    var URLRequest: NSMutableURLRequest {
+    func asURLRequest() -> URLRequest {
         let (path, method, headers, data, parameters) = self.components
-        let fullURL: NSURL
-        if let URL = NSURL(string: path) where URL.host != nil {
-            fullURL = URL
+        let fullUrl: URL
```
Comment (sberrevoets) [inline Sources/Sonar/APIs/AppleRadarRouter.swift]: Would leave this as `fullURL`

**Comment by sberrevoets** [comment]
:shipit: though comments are still old format ¯\_(ツ)_/¯

File: Sources/Sonar/APIs/AppleRadarRouter.swift (hunk 1 above)
Comment (keith) [inline Sources/Sonar/APIs/AppleRadarRouter.swift]: fixed

**Comment by keith** [comment]
Crap I have a script for this _somewhere-

**Pull request merged**